RationaleMCP/Hints.md

```diff
@@ -0,0 +1,12 @@
+Hints for generating a release.
+
+Currently there are no automated procedures for generating the revision-history.
+
+For the MCPs it is good to list additional pull requests considered, they can be found using something like:
+```
+-base:master -base:maint/3.4 -base:maint/3.5 merged:>=2000-01-01 -label:MCP0031 -label:MCP0033 
```

Comment:
Why not also suggest a command at the start of the line?

Reply:
Clarified that you do it in GitHub pull request search.
A better alternative is to use the API; I'm working on that.

Reply:
BTW: I couldn't get the API to work, and I would anyway need to check each PR as the names and labels aren't fully reliable. This indicates that using the API will require more planning for the future.

Reply:
… and then what?

Reply:
Those were the ones included in revision-chapter under their respective MCPs (for the merged ones).

It can also be simplified to:
`-base:master -base:maint/ is:merged -label:MCP0031 -label:MCP0033 `
With the explanation that MCP0031 was merged in the previous release, and MCP0033 hasn't been included yet.

However, I believe that if we think of having an external list of changes - or just providing the tools for finding them -  then we should make sure to have labels for each MCP.

Reply:
Having labels for each MCP sounds perfect to me.

Anyway, what started this conversation was a question on a much simpler level: wouldn't the tip be more helpful if it didn't just specify command line options, but also the command to which these options should be passed?

Reply:
Sorry, this change has been very confusing to me. Locally the status is clean - I have pushed my changes, nothing to commit and the work tree is clean, but the preceding line says:

For the MCPs it is good to list additional pull requests considered, they can be found in GitHub's Pull Request list by searching for:

I tried to make another change and push that.

Reply:
OK.  I see that all of this it completely outdated now.  Resolving.